ms/CMakeLists.txt

```diff
@@ -260,6 +261,7 @@ MSSel/MSSelectableTable.h
 MSSel/MSSelection.h
 MSSel/MSSelectionError.h
 MSSel/MSSelectionErrorHandler.h
+MSSel/MSSSpwErrorHanlder.h
```

Comment:
Indeed.  Thanks for finding it (though I don't understand how I was able to build without fixing this typo).  I just pushed the fix.